pkgs/development/python-modules/ratarmountcore/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+, pytestCheckHook
+, indexed-bzip2
+, indexed-gzip
+, indexed-zstd
+, python-xz
+, rapidgzip
+, rarfile
+, zstandard     # Python bindings
+, zstd          # System tool
+}:
+
+buildPythonPackage rec {
+  pname = "ratarmountcore";
+  version = "0.6.0";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "564393eae729a6fba0ed6964dc5a47d1a4aeabedd3b486a6453bb859a4f9c499";
+  };
+
+  propagatedBuildInputs = [ indexed-gzip indexed-bzip2 indexed-zstd python-xz rapidgzip rarfile ];
+
+  pythonImportsCheck = [ "ratarmountcore" ];
+
+  nativeCheckInputs = [ pytestCheckHook zstandard zstd ];
+  # Many tests require sample files that only exist in the git repo and are not shipped with
+  # the source tarball on PyPI. Even the shipped tests are more of a bug than a foresight.
```

Comment:
This would commonly be grounds to fetch from pypi

Reply:
Changed to gitfetch and all tests run after some upstream changes. I'm still not convinced about this. I definitely don't want any normal user to have to run these tests just for installing a package. They take 3 minutes 51 seconds on my system.